packages/react/src/themes/teams-high-contrast/components/Alert/alertVariables.ts

```diff
@@ -0,0 +1,47 @@
+import { SiteVariablesPrepared } from '../../../types'
+
+export interface AlertVariables {
```

Comment:
import `AlertVariables` from `teams` theme instead of declaring a new one.

Reply:
Good catch!